src/tests/account/test_account.cairo

```diff
@@ -71,7 +71,7 @@ fn setup() -> ComponentState {
 }
 
```

Comment:
I think we can use these helpers to avoid repeating the `try_into().unwrap()`.

```
fn TRANSACTION_VERSION() -> felt252 {
    TRANSACTION_VERSION.try_into().unwrap()
}
```

Same applies for `QUERY_VERSION`